src/data/nat/parity.lean

```diff
@@ -138,6 +138,18 @@ begin
   exact @modeq.modeq_mul _ _ 1 _ 1 h₁ h₂
 end
 
+theorem odd_mul : odd (m * n) ↔ odd m ∧ odd n :=
```

Comment:
How about an `odd.mul` version of this, for dot notation and for symmetry with `even.mul`?

Reply:
If I understand you correctly, I have this below as `odd.mul_odd`.

Reply:
Sorry, my comment about symmetry with `even.mul` wasn't right. I meant something with type `odd (m * n) → odd m ∧ odd n`, or even better, two lemmas `odd.left : odd (m * n) → odd m` and for `n`. This way if you have `h : odd (m * n)` you can write `h.left : odd m`.

Reply:
Would `odd.of_mul : odd (m * n) → odd m ∧ odd n` allow `h.of_mul.left` and `h.of_mul.right`?

Reply:
It would, but I don't see any harm in adding the left and right versions directly. Most of the time you'll want one of those, not the conjunction. We can have it all.